shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/create_order/CreateOrderViewModel.kt

```diff
@@ -66,7 +67,11 @@ class CreateOrderViewModel(
         val event = if (addressUiList.isEmpty()) {
             OrderCreationState.Event.OpenCreateAddressEvent
         } else {
-            OrderCreationState.Event.ShowUserAddressListEvent(addressUiList)
+            OrderCreationState.Event.ShowUserAddressListEvent(
+                addressList = addressUiList,
+                selectedUserAddressUuid = orderCreationState.value.deliveryAddress?.uuid
+                    ?: Constants.NOT_SELECTED
```

Comment:
Why we need this constant? Why not use `null`?

Reply:
+